drivers/haptics/drv2605.c

```diff
@@ -600,6 +609,7 @@ static const struct haptics_driver_api drv2605_driver_api = {
 		.feedback_brake_factor = DT_INST_ENUM_IDX(inst, feedback_brake_factor),            \
 		.loop_gain = DT_INST_ENUM_IDX(inst, loop_gain),                                    \
 		.actuator_mode = DT_INST_ENUM_IDX(inst, actuator_mode),                            \
+		.rated_voltage = DRV2605_CALCULATE_VOLTAGE(DT_INST_PROP(inst, vib_rated_mv)),      \
```

Comment:
would use `_mv` in the field name as well (`rated_voltage_mv`), same in the next one

Reply:
After `DRV2605_CALCULATE_VOLTAGE` expands the value is no longer in millivolts. I feel adding the unit suffix is misleading.

Reply:
oh I see, sorry I missed that bit